deps/build.jl

```diff
@@ -7,15 +7,6 @@ if isfile(depsfile)
     rm(depsfile)
 end
 
-function write_depsfile(path)
```

Comment:
Why not use the deps file? its pretty standard

Reply:
Since in this case Xpress will be installed on the users machine and since we are not using `BinaryProvider`, I thought I'd commit the `deps` file to the repository. We will have to update the file manually anytime anything changes. So it's either use the `write_depsfile` function to create a `deps.jl` file on the user's machine or commit it to the repo. I don't feel strongly about this I can revert this change.

Reply:
lets rever for now. to keep it similar to gurobi and cplex.

Reply:
Resolved in https://github.com/JuliaOpt/Xpress.jl/pull/43/commits/733a0770e340d71e0cee8848fe63705eeb5fe28a.